gradle/sdk-registry.gradle

```diff
@@ -58,13 +58,13 @@ task androidJavadocs(type: Javadoc) {
 }
 
 task javadocsJar(type: Jar, dependsOn: dokkaJavadoc) {
-    classifier = 'javadoc'
+    classifier "javadoc"
     from dokkaJavadoc.outputDirectory
 }
 
 task androidSourcesJar(type: Jar) {
-    classifier = 'sources'
-    from android.sourceSets.main.kotlin
+    classifier "sources"
+    from android.sourceSets.main.java.srcDirs
```

Comment:
could you verify the android sources jar is generated correctly with this change?

Reply:
Verified the kotlin dir is picked up correctly for viewport plugin, this should be ok.